### ISSUE TITLE
Audit rules to ensure correct behavior when no `meta` object present

### ISSUE THREAD

**Issue by bmish** [not-an-aardvark/eslint-plugin-eslint-plugin#181]
This is a bit of an edge case but I suspect some rules aren't handling it properly.

**Comment by aladdin-add** [comment]
function-style rules were not recommended (and documented?), not sure it's worth too many efforts to cover it.

and, we already have rule `prefer-object-rule`.

**Comment by bmish** [comment]
I was actually referring to object-style rules that are missing a `meta` object, as below:

```js
module.exports = {
  create() {}
}
```

It's rare that a rule would be left like this, but especially when writing a new rule, the `meta` object might not be added yet, so we want to make sure existing rules don't crash.

**Comment by aladdin-add** [comment]
it should have been reported by rules like `require-meta-*`(type/fixable/...)?
